packages/catapult-client/catapult-client.0.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Network client for catapult, to be paired with catapult-daemon"
+maintainer: ["simon@imandra.ai"]
+authors: ["simon@imandra.ai"]
+license: "MIT"
+homepage: "https://github.com/AestheticIntegration/catapult"
+bug-reports: "https://github.com/AestheticIntegration/catapult.git/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "catapult" {= version}
+  "odoc" {with-doc}
+  "zmq" {>= "5.0"}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
Can you replace `pinned` by `dev` into your packages?

Reply:
I forget, how do you get dune to fix that? I have dune 3, I'm surprised :/